content/en/docs/languages/ruby/libraries.md

```diff
@@ -98,6 +114,32 @@ OpenTelemetry::SDK.configure do |c|
 end
 ```
 
+#### Configuring specific instrumentation libraries with environment variable
+
+You can also define the option for specific instrumentation libraries using
+environment variables. By convention, the environment variable will be the name
+of the instrumentation, with uppercase, `::` replaced by underscores,
+`OPENTELEMETRY` shortened to `OTEL_{LANG}`, and `_CONFIG_OPTS` appended.
+
+For example, the environment variable name for
+OpenTelemetry::Instrumentation::Faraday is
+`OTEL_RUBY_INSTRUMENTATION_FARADAY_CONFIG_OPTS`. A value of
+`peer_service=new_service;span_kind=client` overrides the options set from
+[previous section](#configuring-specific-instrumentation-libraries) for Faraday.
+
+```bash
+export OTEL_RUBY_INSTRUMENTATION_FARADAY_CONFIG_OPTS="peer_service=new_service;span_kind=client"
+```
+
+| Data Type | Value                      | Example          |
+| --------- | -------------------------- | ---------------- |
+| Array     | string with `,` separation | `option=a,b,c,d` |
+| Boolean   | true/false                 | `option=true`    |
+| Integer   | string                     | `option=string`  |
+| String    | string                     | `option=string`  |
+| Enum      | string                     | `option=string`  |
```

Comment:
It looks like string values need quotes, so we should add those to the example. Also, does an array value require quotes or brackets?
```suggestion
| Integer   | string                     | `option="string"`  |
| String    | string                     | `option="string"`  |
| Enum      | string                     | `option="string"`  |
```

Reply:
I think In ruby, it’s not necessary for strings to be enclosed in quotes.

> does an array value require quotes or brackets

No, for array, it will be something like: `export XXX_CONFIG_OPTS="example_array=a,b,c,d;"

Reply:
Sorry for the confusing.

I had the example like `export XXX_CONFIG_OPTS="example_array=a,b,c,d;"` to avoid the confusion like abc=def=ghi (double equal sign confusing key=value e.g. key: abc=def, value: ghi);
So I want to enclose the value with quote. 
But I think for bash it's ok to have something like abc=def=ghi and key, value will be parsed correctly (key: abc, value: def=ghi).

Therefore, yes, the quotes are optional.

Reply:
Thanks, @xuan-cao-swi! I'm resolving this conversation.